lace/emulator/pnd_archive.py

```diff
@@ -11,362 +11,530 @@
 from lace.cosmo import fit_linP
 
 
+def get_sim_option_list(sim_suite):
+    """
+    Get the simulation option list based on the specified simulation suite.
+
+    Args:
+        sim_suite (str): Name of the simulation suite.
+
+    Returns:
+        list: List of simulation options available for the specified simulation suite.
+
+    """
+    if (
+        (sim_suite == "Pedersen21")
+        | (sim_suite == "Cabayol23")
+        | (sim_suite == "768_768")
+    ):
```

Comment:
I guess you could have here used an "assert", since I think the code will crash if it does not enter this "if" loop. Line 55 will try to return a bunch of variables not defined.

Reply:
Check out line 120. In the beginning of the class, I check whether sim_suite is within the available list, and if not, the program ends. I used ifs instead of asserts because I was thinking of modifying the bookkeeping to read the Nyx simulations. Having said that, I am happy to use assert for the time being

Reply:
Ok, let's leave it as is, and worry about Nyx in a future PR soon.